tools/dump_all_tables_data_only.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
this tool should be expanded to include a load database and a dump database functionality